docs/box.rst

```diff
@@ -0,0 +1,80 @@
+Box
+=================
+
+********
+Overview
+********
+
+`Box <https://box.com>`_ is a commercial file sharing service similar to Dropbox and Google Drive. The current Parsons API supports listing files and folders, creating and deleting folders, and uploading/downloading Parsons tables as either CSV or JSON files.
+
+.. note::
+  Authentication
+    This connector requires authentication credentials for a Box business-level account. Information on setting up those credentials can be found at the `Box Custom App Quick Start page <https://developer.box.com/guides/applications/custom-apps/oauth2-setup/>`_.
+
+**********
+Quickstart
+**********
+
+This class requires credentials in the form three strings to be either passed into the constructor or made available as environment variables:
+
+.. code-block:: bash
+
+  export BOX_CLIENT_ID=dp4rqi0cz5qckz361fziavdtdwxz
```

Comment:
Are these fake keys?

Reply:
Yes, fake keys. Shall I add a note in the doc saying "These are fake - go generate your own"?